robyn/__main__.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+
+project_name = input("Enter the project name: ")
+project_dir = input("Enter the name of the project directory: ")
+
+
+def create(project_name, project_dir):
```

Comment:
@Oluwaseun241 , I didn't mean this 😅 

The code would look like the following

```python3
import click
import os

@click.command()
def create():
  project_name = input("Enter the project name: ")
  project_dir = input("Enter the name of the project directory: ")
  print(f"Creating a new Robyn project '{project_name}' in '{project_dir}'...")

    # Create a new directory for the project
    project_path = os.path.join(project_dir, project_name)
    os.makedirs(project_path)

    # Create the main application file
    app_file_path = os.path.join(project_path, "app.py")
    with open(app_file_path, "w") as f:
        f.write(
            'from robyn import Robyn\n\napp = Robyn()\n\nif __name__ == "__main__":\n app.run()'
        )

    print(f"New Robyn project '{project_name}' created in '{project_dir}' ")
    
if __name__=="__main__":
   create()
```

Reply:
I think using click is not that efficient again since we're not using the click.option